### PR TITLE
Packit enablement

### PR DESCRIPTION
**Pull request by pemensik**
Simple changes to prepare packit building

### CODE REVIEW

**Comment by pemensik** [comment]
I have created COPR build for this repository, but without automation watching commits.

File: tests/build_package.sh [hunk 1]
```diff
@@ -3,7 +3,6 @@
 set -e
 
 python3 setup.py sdist -d ./distribution
-cp ./LICENSE distribution/LICENSE
```
Comment (TomasKorbar) [inline tests/build_package.sh:6]: Adding LICENSE symlink is a cool trick! Just needed this change to not break the testsuite.

**Review by TomasKorbar — CHANGES_REQUESTED** [review]
Review finished. When my comments are resolved please squash the commits.

File: .copr/Makefile [hunk 2]
```diff
@@ -0,0 +1,5 @@
+outdir?=.
+root=$(shell git rev-parse --show-toplevel)
+srpm:
+	rpm -q git-core packit || dnf install -y git-core packit
+	if test -n "$(root)"; then cd $(root); fi && pwd && packit srpm --output=$(outdir)
```
[the remark this answers is not in the document]
Comment (pemensik) [reply .copr/Makefile:5]: But I have empty Subdirectory in copr project empty in the end, so yes, this should not be necessary anymore. I expect official packit enablement might be different anyway.

[the remark this answers is not in the document]
Comment (TomasKorbar) [reply .copr/Makefile:5]: Alright, so accept the change please and resolve the conversation.

File: tests/build_package.sh (hunk 1 above)
Comment (TomasKorbar) [reply tests/build_package.sh:6]: Alright. That sounds like you accept the change so i will resolve this conversation.

**Review by TomasKorbar — APPROVED** [review]

**Pull request merged**